### ISSUE TITLE
[3.0.0] Document breaking changes

### ISSUE THREAD

**Issue by davidjb** [eddiewebb/circleci-queue#120]
### Orb version

3.0.0+

### What happened

Updating to 3.1.1, there are number of parameter changes that break previous usage and require delving into the code (https://github.com/eddiewebb/circleci-queue/commit/4c639c89a6c883d00a16aec2a334bdce1ffff0f9) to determine what they are.

### Expected behavior

Update the changelog or other documentation with the specifics of the breaking change - e.g. `my-pipeline` is now required and various parameters have changed their names (e.g. `time` is now `max-wait-time`_. Even if the changelog mentioned these changes in general linked someone off to the specific commit, then that would make updating easier.

**Comment by titanium-cranium** [comment]
+1

The user guide also needs updating, a single argument is no longer correct.  Help determining how to find the pipeline number would also be appreciated.

![Screenshot 2024-03-05 at 11 40 13 AM](https://github.com/eddiewebb/circleci-queue/assets/15062524/e645520e-3718-42a9-9d46-862f0a1ed2a3)

**Comment by davidjb** [comment]
I was slightly puzzled as to this being closed. After digging around a bit, I discovered that this orb is deprecated and CircleCI now has native support for equivalent behaviour. Details are in the published description at  https://circleci.com/developer/orbs/orb/eddiewebb/queue. 

I imagine that’s the gist of it, @eddiewebb?

The replacement `serial-group` feature is detailed at https://circleci.com/docs/configuration-reference/#serial-group and CircleCI official announcement at https://circleci.com/changelog/avoid-commit-conflicts-in-your-pipelines-with-serial-jobs/.

**Comment by eddiewebb** [comment]
Yes! Sorry for the confusion, that's exactly the reason for closing this
and all open items. I left the one open issue sharing details of the
deprecation.
